polaris-terraform/ui-terraform/nginx.conf

```diff
@@ -41,6 +41,11 @@ http {
 
       proxy_pass ${ENDPOINT_HTTP_PROTOCOL}://${UPSTREAM_CMS_DOMAIN_NAME};
     }
+
```

Comment:
I've just moved this up a bit to be next to the CMS classic (which is the default) rule